api/v2beta2/snapshot_types.go

```diff
@@ -156,6 +156,9 @@ type Snapshot struct {
 	// run by the controller.
 	// +optional
 	TestHooks *map[string]*TestHookStatus `json:"testHooks,omitempty"`
+	// OciDigest is the digest of the OCI artifact associated with the release.
+	// +optional
+	OciDigest string `json:"ociDigest,omitempty"`
```

Comment:
This and the last applied value must contain the SHA type instead of like now, just the hex.

Also please use `OCI` instead of `Oci`